### DISCUSSION TITLE
ros1 bag on remote server

### DISCUSSION THREAD

**Discussion by xli4217** [foxglove/community#48] [Q&A]
Hi, I was wondering if there is a way to access ros1 bags on a remote server that would normally access via ssh. Thanks.

**Comment by amacneil** [comment]
Not currently. We do have the ability to stream bags over http, so you could potentially either expose an http/https server, or you could tunnel an http server over ssh.

For http streaming to work you need to enable "range requests" in your http configuration. For example, you can see an nginx config in https://github.com/foxglove/studio/issues/2651.

@esthersweon do we have any documentation around range requests? Might be good to provide some common examples in our docs or a blog post.

**Comment by rogue-shadowdancer** [comment]
If you just want to use bag remotely and do not have to use ssh, maybe this will help you: https://github.com/cruise-automation/webviz/issues/247

**Reply by amacneil** [reply]
Yes, webviz and foxglove both support loading remote bags over http(s). But not ssh.
